tests/unit/test_events.py

```diff
@@ -181,6 +188,21 @@ def test_parse_release(self, release):
         assert event_object.tag_name == "0.0.1"
         assert event_object.project_url == "https://github.com/Codertocat/Hello-World"
 
+    def test_parse_mr(self, merge_request):
```

Comment:
Thanks for adding a test for the parsing. I have some more in mind..;)

- Does pushing new commits to the merge-requests triggers different action? Can we have tests for both situations?
- Can you please also tests for `project` and `package_config`?
  - Look at the similar tests in this file.
- Can you please add the database tests?
  - They are placed here: https://github.com/packit-service/packit-service/blob/master/tests_requre/database/test_events.py
  - Especially, we want to tests `db_trigger` and `project` properties. (Get inspired by other tests there.)

Reply:
> * Does pushing new commits to the merge-requests triggers different action? Can we have tests for both situations?

Pushing new commit to merge-request adds ```action: "update"``` to the event, I have made a new test for that.

> * Can you please add the database tests?

Added the database tests